src/wl-copy.c

```diff
@@ -58,8 +58,12 @@ static void did_set_selection_callback(struct copy_action *copy_action) {
          * so the stdout file descriptor
          * isn't kept alive. See issue #100
          */
-        int output_fds = open("/dev/null", O_WRONLY);
-        dup2(output_fds, STDOUT_FILENO);
+        int devnull = open("/dev/null", O_WRONLY);
+        if (devnull >= 0) {
+            dup2(devnull, STDOUT_FILENO);
+        }
+        fclose(stdout);
+        fclose(stdin);
```

Comment:
Almost perfect! Except...
* Open `/dev/null` for both reading and writing
* Dup it to both stdin and stdout
* Close the fd after that
* *If opening `/dev/null` failed* (not unconditionally!), then close stdin/stdout (with plain `close()`, don't bother with `fclose()`)

Also, please update the comment to mention both stdin and stdout. And I guess remove the issue reference — I don't do that elsewhere. Squash the commits, and make sure the commit message lists the full URL of the issue, like this: `https://github.com/bugaevc/wl-clipboard/issues/100`, not like this: `#100` (though GitHub collapses it into #100 anyway, so I don't know, perhaps you already have it the right way)